chatbot/core/aiml/parse_aiml.php

```diff
@@ -212,7 +212,7 @@ function parse_matched_aiml($convoArr, $type = "normal", $aiml_pattern = '')
     //which debug mode?
     runDebug(__FILE__, __FUNCTION__, __LINE__, "Run the aiml parse in $type mode (normal or srai)", 3);
 
-    $aiml_pattern = ($aiml_pattern == '') ? $convoArr['aiml']['pattern'] : $aiml_pattern;
```

Comment:
Your change is is the same like before. My suggestion makes sense and works perfectly. Here what happend before: (`$aiml_pattern` is not empty on srai calls.)
```text
compairing patterns xtest person test and xtest person test
```
after my change
```text
compairing patterns XTEST PERSON (.*)? and XTEST PERSON * 
```
I could not find any errors so far. It works like before only.